clusters/app.ci/supplemental-ci-images/origin-stable/release.yaml

```diff
@@ -29,6 +29,10 @@ spec:
     from: {"kind":"DockerImage", "name":"registry.ci.openshift.org/ci/origin-release:rhel-7-golang-1.11"}
     referencePolicy: {"type": "Local"}
     importPolicy: {"scheduled": true}
+  - name: golang-1.22
+    from: {"kind":"DockerImage", "name":"registry.ci.openshift.org/ci/origin-release:golang-1.22"}
```

Comment:
Above image is centos8 based which is EOL.
You can use `registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17`

Reply:
It should be fine for now, but you will have to keep updating once centos9 is EOL. Suggesting better use rhel image to avoid this in future. :)